doc/api/process.md

```diff
@@ -1186,7 +1186,8 @@ added: v0.1.100
 * {string}
 
 The `process.execPath` property returns the absolute pathname of the executable
-that started the Node.js process.
+that started the Node.js process. Symbolic links if any, will be resolved
```

Comment:
```suggestion
that started the Node.js process. Symbolic links, if any, will be resolved
```

Reply:
In addition to the comma, I'd prefer that we keep things in present tense if there's a choice:

```suggestion
that started the Node.js process. Symbolic links, if any, are resolved
```

Reply:
@addaleax and @Trott - thanks for the review, I have followed your suggestion. Please have a look.